Zend/zend_errors.h

```diff
@@ -39,7 +39,7 @@
 /* Indicates that this usually fatal error should not result in a bailout */
```

Comment:
Add a TODO Comment about the `#define E_STRICT` to remove this definitely in PHP 9?